lib/bolt/outputter/human.rb

```diff
@@ -310,7 +310,7 @@ def format_table(results, padding_left = 0, padding_right = 3)
         )
       end
 
-      def print_tasks(tasks, modulepath)
+      def print_tasks(tasks:, modulepath:)
```

Comment:
What is the difference here? I presume it has something to do with default values?

Reply:
This was updated to use keyword arguments: https://medium.com/@tyler_brewer2/utilizing-ruby-keyword-arguments-462ba8366483

A few of the benefits are we now explicitly label what the arguments are, position doesn't matter, and we can easily add and remove new args in the future.